### PR TITLE
UI AGPL license consent

### PR DESCRIPTION
**Pull request by prakashsvmx**
Fixes #https://github.com/miniohq/engineering/issues/911
How does it look?:

image

The acknowledgement is stored in local storage key. if it is cleared, it would prompt again.

### CODE REVIEW

**Review by oscarocastellanos — APPROVED** [review]
LGTM

File: portal-ui/src/screens/Console/License/LicenseConsentModal.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+const LicenseConsentModal = ({
+  isOpen,
+  onClose,
+}: {
+  onClose: () => void;
+  isOpen: boolean;
+}) => {
+  const recordAgplConsent = () => {
+    setLicenseConsent(); //to Local storage.
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/License/LicenseConsentModal.tsx:31]: I think we should hide the red dot in the menu once the user acknowledge this

Comment (bexsoft) [inline portal-ui/src/screens/Console/License/LicenseConsentModal.tsx:31]: I think we should hide the red dot in the menu once the user acknowledge this

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by reivaj05 — APPROVED** [review]

**Comment by cniackz** [comment]
Screen Shot 2022-07-05 at 12 13 08 PM

Yeap, I can see the pop-up from here.

**Review by cniackz — APPROVED** [review]
Looks good to me, it appears once if acknowledged

**Pull request merged**